apps/wallet/src/engine/runners/solana/tokens.ts

```diff
@@ -34,10 +34,13 @@ export const getTokenDocumentsOnChain = async (
 
 	const tokens = await Promise.all(tokenPromises);
 
-	const quotes = await getTokenQuotes(tokens);
+	const quotes = await getTokenQuotes(
+		tokens.map((token) => ({ address: token.mint, network: token.network })),
+	);
 
 	for (const item of tokens) {
-		item.account.quotes = quotes[makeHashId(item)].quotes;
+		item.quotes =
+			quotes[makeHashId({ address: item.mint, network: item.network })].quotes;
```

Comment:
if possible, check if we can refactor `makeHashId` (if don't use it else where) - otherwise, use this:
```typescript
item.quotes = quotes[makeHashId(tokenToArg(item))].quotes;
```